site/ko/blog/new-in-devtools-94/index.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: "layouts/blog-post.njk"
+title: "What's New In DevTools (Chrome 94)"
+authors:
+  - jecelynyeen
+date: 2021-08-24
+updated: 2021-08-24
+description:
+  "Use DevTools in your preferred language, new Nest Hub devices, new CSS container queries badge and more."
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/js1uKEulNBGj0PoJkqZq.jpg'
```

Comment:
change image to `image/dPDCek3EhZgLQPGtEG3y0fTn4v82/BTDUZobWsxTQFkQo9nfn.jpg`

Reply:
Would you mind sharing a URL? Sorry I don't know how to see the image.

Reply:
https://wd.imgix.net/image/dPDCek3EhZgLQPGtEG3y0fTn4v82/BTDUZobWsxTQFkQo9nfn.jpg?auto=format&w=1600

Reply:
Thanks! LGTM